src/main/java/com/github/davidmoten/rx2/flowable/Transformers.java

```diff
@@ -4,7 +4,13 @@
 import java.util.Comparator;
 import java.util.List;
 import java.util.concurrent.Callable;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
 
+import com.github.davidmoten.guavamini.Optional;
+import com.github.davidmoten.rx2.internal.flowable.*;
```

Comment:
In fact I often find the wildcard imports frustrating when I'm browsing source in github. I look at a class reference and don't know where to find it because it could be part of multiple wildcard imports.